mypy/server/update.py

```diff
@@ -402,6 +405,24 @@ def find_unloaded_deps(manager: BuildManager, graph: Dict[str, State],
     return unloaded
 
 
+def ensure_deps_loaded(module: str,
+                       deps: Dict[str, Set[str]], graph: Dict[str, State]) -> None:
+    """Ensure that the dependencies on a module are loaded.
```

Comment:
It's worth mentioning in the docstring what state is mutated. In particular, is `deps` modified?